cfme/tests/services/test_provision_stack.py

```diff
@@ -153,7 +178,6 @@ def dialog(provider, provisioning):
                                      template_name=fauxfactory.gen_alphanumeric())
     return dialog_name, template
 
-
```

Comment:
I think cfme requires two lines and wrapanapi only one.  But I think you're right.